src/UKHO.S100PermitService.Common/Encryption/AesEncryption.cs

```diff
@@ -25,6 +38,19 @@ public string Decrypt(string hexString, string hexKey)
             return PerformCryptography(hexString, decrypt);
         }
 
+        /// <summary>
+        /// Get encrypted data.
+        /// </summary>
+        /// <remarks>
+        /// Encrypt data using secret key.
+        /// Advanced Encryption Standard (AES) block cipher algorithm is used.This is a symmetric-key algorithm. This means that the same key is used for encryption and decryption.
+        /// Cipher Block Chaining(CBC) mode of operation and No padding is used.
+        /// For S-100 size of data and secret key is fixed to 128 bits (32 characters) hexadecimal digits, if validation fails then AesEncryptionException exception handler will be trigger.
```

Comment:
AesEncryptionException exception will be thrown. Please update elsewhere.

Reply:
Resolved